src/main/java/org/opentripplanner/routing/api/response/RoutingErrorCode.java

```diff
@@ -0,0 +1,25 @@
+package org.opentripplanner.routing.api.response;
+
+public enum RoutingErrorCode {
+  SYSTEM_ERROR,
+
+  /**
+   * Location was found, but it was located outside the street network.
+   */
+  OUTSIDE_BOUNDS,
+
+  /**
+   * The time provided was outside the available service period.
+   */
+  OUTSIDE_SERVICE_PERIOD,
+
+  /**
+   * The location could not be matched to a valid id or coordinate.
+   */
+  LOCATION_NOT_FOUND,
```

Comment:
I think we need to be more specific: FROM_PLACE_NOT_FOUND, TO_PLACE_NOT_FOUND (INTERMEDIATE_PLACE_NOT_FOUND when supporting via search)

Reply:
The idea here was to use the paramValue in RoutingError to specify which place was not found.

Reply:
Maybe it is better to add another enum to report back the incorrect input field. Then that could also be used with the next error code.

Reply:
Changed the paramValue to an enum, and then it is checked in the PlannerErrorMapper and mapped to the correct Message.